ansible/index.md

```diff
@@ -50,6 +50,17 @@ These are maximums. Power consumption varies greatly based on which USB controll
 - 19ma @ -12V
 - No +5v required
 
+## Panel details
+
+![Panel Overlay](images/ansible_CLEAN.png)
+
+- 4 Trigger outputs (TR 1-4, voltage range, slew rate, sample rate, bit rate?)
+- 4 continuous CV outputs (CV 1-4, voltage range, slew rate, sample rate, bit rate)
+- 2 KEY buttons
+- 2 CV inputs (IN 1-2, voltage range, sample rate, bit rate)
+- 1 USB-A port
+- 1 MODE button
```

Comment:
I realize the docs use the word "key" rather than "button" to describe these ... interaction elements, however since there are already two which are labeled KEY, it seems a bit confusing to call them "KEY keys", or to have "KEY buttons" but "MODE key"

Thoughts?

Reply:
@rknLA I think I like KEY, as it's a bit more canonical -- we also use it for norns. I think you're right, though, that this needs a bit of breakdown and thought.

w/r/t MODE: I wonder if *PRESET* isn't the more important identifier for that key? maybe it'd be better to include a note in the panel breakout that makes this explicit based on action:

`PRESET (press) / MODE (hold)`

Reply:
action-specific makes sense. The main reason I changed this to MODE on line 78 is because down around line 96 it’s referred to as the MODE button often, and i didn’t see where it’s referred to as preset in this document (it’s only in the docs for the other modules that it’s referred to as a preset button/key, and since the other modules are OOP, maybe other folks won’t see or notice the convention from those docs)

KEY (all caps) is referenced in the panel image, so keeping that nomenclature makes sense, but whether to call them keys (lowercase) or buttons is confusing to me.

2 KEY buttons and 1 PRESET/MODE key?

Reply:
mmm. totally hear you. to confuse things further, all the panel images in the app docs call the `PRESET`/`MODE` key `PRESET`/`APPLICATION`. lol. I'll spend some time with the modular docs this week, apologies for getting lost in the grid docs the last few cycles!

as far as this page goes, I wonder if is a good guiding principle:
>Output and input functionality vary according to which application is running.

...which also applies to the KEYs.

either way, really great notes, thank you Kevin <3333